internal/metarepos/report_collector_test.go

```diff
@@ -494,6 +495,55 @@ func TestReport(t *testing.T) {
 	})
 }
 
+func TestReportIgnore(t *testing.T) {
+	Convey("ReportCollector should collect report from registered storage node", t, func() {
+		a := NewDummyStorageNodeClientFactory(1, true)
+		mr := NewDummyMetadataRepository(a)
+
+		logger, _ := zap.NewDevelopment()
+		reportCollector := NewReportCollector(mr, DefaultRPCTimeout, newNopTelmetryStub(), logger)
+		reportCollector.Run() //nolint:errcheck,revive // TODO:: Handle an error returned.
+		defer reportCollector.Close()
+
+		// Register Invalid StorageNodeID to set InvalidLogStreamID
+		snID := types.MinStorageNodeID
```

Comment:
Could you check this comment? There is no code to register InvalidStorageNodeID.

Reply:
deleted